isic/login/models.py

```diff
@@ -1,16 +1,24 @@
 import logging
 
+from django.conf import settings
 from django.contrib.auth.models import User
 from django.core.validators import RegexValidator
 from django.db import models
 from django.db.models.signals import post_save
 from django.dispatch import receiver
+from hashids import Hashids
 
 from isic.core.constants import MONGO_ID_REGEX
 
 logger = logging.getLogger(__name__)
 
 
+def get_hashid_hasher():
```

Comment:
This would be better abstraction as:
```suggestion
def get_hashid(value: int) -> str:
```